extensions/objects/templates/FilepathEditor.js

```diff
@@ -161,6 +161,12 @@ oppia.directive('filepathEditor', [
           $scope.filepaths = response.data.filepaths;
           $scope.filepathsLoaded = true;
         });
+        $scope.width = null;
```

Comment:
I think option (b) sounds good. That way you can have other tools for editing the image as well.
What do you meant by "adjust the preview in the RTE" in option (a)? And did you mean going back to what we had in the beginning with the input box for width and height in the modal without a preview?